docs/new_flux_ideas_and_guide.md

```diff
@@ -0,0 +1,985 @@
+underlying time series would be represented by the combination of `location`, `state`, and the specific field. 
+Results are ordered by default by the columns of the tags and their values and then by time. So we can see that 
+`coyote_creek`, `CA` rows are together in time ascending order.
+
+We'll show later how to specify to the graphing engine what represents an individual time series that you want 
+visualized.
+
+## Basic Queries
+This section will explore some basic queries in Flux to highlight the syntax and a few of the basic concepts of
+the language. For these examples, we'll assume a dataset that has a schema like the one seen above.
+
+### Introduction to basics and getting the last value
+In this example, we'll get the last value of `surface_degrees` for every series in the `h2o_temperature` 
+measurement in the `my_data` bucket. Flux has a shorthand syntax for the common operation of selecting data from 
+a bucket with given criteria for a given time range. Let's look at that first for this example:
+
+```
+@my_data.h2o_temperature{surface_degrees, time}
+```
+
+The `@` symbol at the beginning indicates that we're using the shorthand syntax and that the next set of characters
+will identify the bucket name. After that we have a `.` separator, followed by the measurement name of 
+`h2o_temperature`. Immediately following that we have the **Column Filter Predicate** syntax (the curly braces
+`{` and `}` and what's inside of them) to indicate which columns we want in the result. The identifiers can
+also be wrapped in double quotes, which you'll need to use if they contain spaces:
+
+```
+@"my_data"."h2o_temperature"{"surface_degrees", "time"}
+```
+
+Given the previous data set, this query would produce a result like this: 
+
+| surface_degrees | time |
+| --- | --- |
+| 50.2 | 1600756160 |
+| 54.7 | 1600756160 |
+| 63.6 | 1600756160 |
+
+Since we didn't specify a time range in the query, we get the most recent data point for each series. How far 
+back in time the database looks for a recent value is dependent on the setup of the backend InfluxDB system you 
+are working with (this will typically be less than a few hours). We'll show how to specify explicitly how far 
+back to look for the most recent value in a moment.
+
+In this result set we have three rows, but we can't see their series information (i.e. the tags) because we 
+didn't select them. Selecting them is fairly easy:
+
+```
+@my_data.h2o_temperature{location, state, surface_degrees, time}
+```
+
+Would return the last value of surface_degrees in each series:
+
+| location | state | surface_degrees | time |
+| --- | --- | --- | --- |
+| coyote_creek | CA | 50.2 | 1600756160 |
+| puget_sound | WA | 54.7 | 1600756160 |
+| santa_monica | CA | 63.6 | 1600756160 |
+
+
+In Flux, a tag key and a field key are both called columns. This means that tag keys and field keys must
```

Comment:
I'm not sure how helpful this detail is for users getting started IMO. I wasn't thinking about unique names here.